lib/token-verifier/WP_Auth0_IdTokenVerifier.php

```diff
@@ -0,0 +1,265 @@
+<?php
```

Comment:
[See this diff](https://github.com/auth0/auth0-PHP/pull/401/files#diff-3871126b680fac12ec0aa3124b74e102) for changes from approved PHP SDK ones.